js_modules/dagit/packages/core/src/overview/OverviewSensorsRoot.tsx

```diff
@@ -137,7 +204,22 @@ export const OverviewSensorsRoot = () => {
       );
     }
 
-    return <OverviewSensorTable repos={filteredBySearch} />;
+    return (
+      <OverviewSensorTable
+        headerCheckbox={
+          viewerHasAnyInstigationPermission ? (
+            <CheckAllBox
```

Comment:
oo did not know we had this component. this will be useful with the new run page bulk actions.

Reply:
Just created it with the Schedules bulk actions! :)